$OpenBSD: patch-configure,v 1.24 2012/01/25 11:30:41 ajacoutot Exp $

Don't do bash pattern substitutions.

Our libdb does not follow upstream naming (libdb.so.XX versus
libdb-5.0.so).

--- configure.orig	Sun Nov 20 15:26:59 2011
+++ configure	Wed Jan 25 12:09:40 2012
@@ -20173,9 +20173,9 @@ $as_echo_n "checking if gobject-introspection is corre
             INTROSPECTION_COMPILER=`$PKG_CONFIG --variable=g_ir_compiler gobject-introspection-1.0`
             INTROSPECTION_GENERATE=`$PKG_CONFIG --variable=g_ir_generate gobject-introspection-1.0`
             INTROSPECTION_GIRDIR=`$PKG_CONFIG --variable=girdir gobject-introspection-1.0`
-            INTROSPECTION_GIRDIR=${INTROSPECTION_GIRDIR/$datadir/\$(datadir)}
+            INTROSPECTION_GIRDIR=`echo ${INTROSPECTION_GIRDIR} | sed -e 's,$datadir,\$\(datadir\),g'`
             INTROSPECTION_TYPELIBDIR="$($PKG_CONFIG --variable=typelibdir gobject-introspection-1.0)"
-            INTROSPECTION_TYPELIBDIR=${INTROSPECTION_TYPELIBDIR/$libdir/\$(libdir)}
+            INTROSPECTION_TYPELIBDIR=`echo ${INTROSPECTION_TYPELIBDIR} | sed -e 's,$libdir,\$\($libdir\),g'`
             INTROSPECTION_CFLAGS=`$PKG_CONFIG --cflags gobject-introspection-1.0`
             INTROSPECTION_LIBS=`$PKG_CONFIG --libs gobject-introspection-1.0`
             INTROSPECTION_MAKEFILE=$introspection_makefile
@@ -21023,9 +21023,9 @@ $as_echo_n "checking for Berkeley DB files in $d... " 
 		        db_libfile="$d/$bdb_loclibdir/libdb$sversion$bdb_libext"
         	        try_headers="db.h db$version/db.h db$sversion/db.h"
 		    else
-		        db_libfilename="libdb-$version$bdb_libext"
-        	        db_lib="-ldb-$version"
-		        db_libfile="$d/$bdb_loclibdir/libdb-$version$bdb_libext"
+		        db_libfilename="libdb$bdb_libext.${LIBdb_VERSION}"
+        	        db_lib="-ldb"
+		        db_libfile="$d/$bdb_loclibdir/libdb$bdb_libext.${LIBdb_VERSION}"
         	        try_headers="db.h db$version/db.h db$sversion/db.h"
     		    fi
 		fi
